src/activateDaffodilDebug.ts

```diff
@@ -51,10 +66,9 @@ export function activateDaffodilDebug(context: vscode.ExtensionContext, factory?
 		})
 	);
 
-	context.subscriptions.push(vscode.commands.registerCommand('extension.dfdl-debug.getProgramName', config => {
+	context.subscriptions.push(vscode.commands.registerCommand('extension.dfdl-debug.getDataName', config => {
 		return vscode.window.showInputBox({
-			placeHolder: "Please enter the name of a markdown file in the workspace folder",
-			value: "readme.md"
+			placeHolder: "Enter the path of the data file to be used:"
```

Comment:
Is there any way to add a file picker to this? Typing full paths could be tedious if files are not in the workspace, which most likely they won't be.